ExeterTeam25/group-software-project
# Forget password

To Test:
- pip install python-decouple
- in the group_project1 directory create a file called .env
- in .env add the information that I dm to you (on snapchat)
- create a user with a valid email (Use your uni email)
- logout
- go to login page
- Click forgot password?
- enter the valid email for that user
- you should receive an email from exeterteam25@gmail.com (check spam/junk)
- click the link in the email
- change the password to something different
- try and login with the new password

Details:
- added functionality to allow the user to change their password
- User will be authenticated with email
- created an gmail for the
